data-mongodb/src/test/java/io/micronaut/data/document/mongodb/repositories/ElementRowRepository.java

```diff
@@ -0,0 +1,62 @@
+package io.micronaut.data.document.mongodb.repositories;
+
+import io.micronaut.data.document.mongodb.entities.ElementRow;
+import io.micronaut.data.mongodb.annotation.MongoAggregateQuery;
+import io.micronaut.data.mongodb.annotation.MongoRepository;
+import io.micronaut.data.repository.CrudRepository;
+import io.micronaut.serde.annotation.Serdeable;
+import org.bson.types.ObjectId;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+@MongoRepository
+public interface ElementRowRepository extends CrudRepository<ElementRow, ObjectId> {
+
+    @MongoAggregateQuery("[{ $match: {$and: [{ eventId: :eventId}, {rowState: :rowState}] } },"
```

Comment:
You can use `"""` to avoid escaping

Reply:
Thanks, I changed it to single quotes and removed double quotes where not needed so it is a bit more readable.